chat.html

```diff
@@ -13,9 +13,10 @@
           MessageCanvasTrayContent: "<p>Ο σύμβουλος εγκατέλειψε τη συνομιλία. Σας ευχαριστούμε που επικοινωνήσατε. Παρακαλώ επικοινωνήστε μαζί μας ξανά εάν χρειάζεστε περισσότερη βοήθεια.</p>"
         },
         'en-US': {
-          BotGreeting: "How can I help?",
           WelcomeMessage: "Welcome to Toronto Line!",
           MessageCanvasTrayContent: "<p>The counselor has left the chat. Thank you for reaching out. Please contact us again if you need more help.</p>",
+          MessageInputDisabledReasonHold: "Please hold for a counselor.",
```

Comment:
Just to double-check, this message will show up in English for people with other languages, right?  (Per Slack conversation, that's fine.)

Reply:
Yes, the translation takes place after we have already set the default strings in English.